ONSdigital/eq-author-app
# Fix main and right panel not scrolling

### What is the context of this PR?

The scrollPane that was sending the panels back to the top was removed. Have re-added it to editor layout to replicate the same functionality.

### How to review

1. Create questionnaire
2. Add a bunch of sections and questions 
3. Switch pages
4. Left nav should stay and main and right should go to the top
